CONTRIBUTING.md

```diff
@@ -4,19 +4,21 @@
 
 ### Installing
 
-Once the repo has been cloned, install Lerna:
+Once the repo has been cloned, install Lerna
```

Comment:
I don't think we should remove the colon, grammatically speaking.

Reply:
I disagree. In all previous documentation, we went away from using these.

Reply:
In our documentation site we use them: https://www.fast.design/docs/en/contributing/install

Reply:
For consistency with existing FAST docs.  Though, I don't agree they should be there either.